docs/sphinx/source/whatsnew/v0.11.3.rst

```diff
@@ -12,12 +12,24 @@ Enhancements
 ~~~~~~~~~~~~
 
 
+Bug Fixes
+~~~~~~~~~
+* Ensure proper tz and pytz types in pvlib.location.Location. tz becomes the
+  single source of time-zone truth, is the single time-zone setter interface,
+  and the getter consistently returns an IANA string. datetime.tzinfo
+  subclasses are fully supported in tz setter. pytz attribute becomes read
+  only. (:issue:`2340`, :pull:`2341`)
```

Comment:
```suggestion
* Ensure proper tz and pytz types in pvlib.location.Location.
```

Ensuring types is a bug fix. The remainder describes an API change that should be announced elsewhere. Not really an enhancement, so maybe we put the rest of this note in the "Breaking changes" section, because one can no longer set a value for `.pytz`.

Reply:
@cwhanse Lmk if my `whatsnew` change for this looks ok.